### PR TITLE
Feature/56144 home screen

### PR DESCRIPTION
**Pull request by dshire**
Added basic home screen, colors, some new components and cleaned up some older components

task https://cognigy.visualstudio.com/Cognigy.AI/_workitems/edit/56144
part of US https://cognigy.visualstudio.com/Cognigy.AI/_workitems/edit/55708

### CODE REVIEW

File: src/webchat-ui/components/presentational/Button.tsx [hunk 1]
```diff
@@ -12,46 +13,52 @@ export default styled.button<IColorProps>(({ color, theme }) => {
     }
 
     if (color === 'primary') {
```
Comment (fabclj) [inline src/webchat-ui/components/presentational/Button.tsx:15]: Here I don't get why we set secondaryColor if when color is equal to 'primary'

**Comment by fabclj** [comment]
the buttons have 16px font-size, but the expected should be 14px
Screenshot from 2023-11-16 10-10-51

File: src/webchat-ui/components/presentational/Button.tsx (hunk 1 above)
Comment (dshire) [reply src/webchat-ui/components/presentational/Button.tsx:15]: According to the design specs, the so called "primary button" is using the so called "secondary colors"

**Comment by dshire** [comment]
> the buttons have 16px font-size, but the expected should be 14px Screenshot from 2023-11-16 10-10-51

fixed

**Review by fabclj — APPROVED** [review]

**Pull request merged**